runtime/moonbase/src/xcm_config.rs

```diff
@@ -328,20 +270,7 @@ impl xcm_executor::Config for XcmExecutorConfig {
 	// When we receive a non-reserve asset, we use AssetManager to fetch how many
 	// units per second we should charge
 	type Trader = (
-		UsingComponents<
```

Comment:
We only have one representation of our SelfReserve token now, so only one UsingComponents